components/picker/themes/spectrum-two.css

```diff
@@ -13,7 +13,7 @@
 
 @container style(--system: spectrum) {
 	.spectrum-Picker {
-		--spectrum-picker-background-color-default: var(--spectrum-gray-100);
```

Comment:
[Looking at the S2 picker file](https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=10528-2695&t=3zNNXsC1Qlr1yGjy-4), `gray-100` _is_ the background color defined for this component. I wouldn't think of this as a bug.

The change to `gray-50` basically makes that picker look more like the S1 picker, but it would have to be reverted once the S2 picker is merged in main. Do we want to keep this correct, `gray-100` in the `spectrum-two.css` theme file?